fixture-menu.js

```diff
@@ -33,5 +33,10 @@ contextMenu({
 (async () => {
 	await app.whenReady();
 
-	await (new BrowserWindow()).loadFile(path.join(__dirname, 'fixture.html'));
+	await (new BrowserWindow({
+		webPreferences: {
+			nodeIntegration: true,
+			spellcheck: true
```

Comment:
The fact that this is needed needs to be documented.

Reply:
Seems like this is needed for the `searchWithGoogle` option being added. Would it be possible to implement a callback for the menu option, and avoid the `require` call? I think its very important to have a workaround for the nodeIntegration requirement, as there is no way we would ever turn that on in our usage, for security reasons.